x-pack/plugins/observability/public/pages/slo_edit/components/timeslice_metric/metric_input.tsx

```diff
@@ -132,7 +132,7 @@ export function MetricInput({
                 selectedOptions={
                   !!indexPattern &&
                   !!field.value &&
-                  AGGREGATION_OPTIONS.some((agg) => agg.value === agg.value)
```

Comment:
Found a bug while referencing the Timeslice Metric implementation

Reply:
Oh! I guess it was ok because there was always a selected field (e.g. isClearable={false}) , but good find nonetheless